server/service/integration_mdm_test.go

```diff
@@ -12805,24 +12813,19 @@ INSERT INTO host_mdm_apple_declarations (
 		return di
 	}
 
-	parseDeclarationResp := func(r *http.Response, expectedBytes []byte) fleet.MDMAppleDDMDeclarationResponse {
+	assertDeclarationResponse := func(r *http.Response, expected fleet.MDMAppleDeclaration) {
 		require.NotNil(t, r)
-		b, err := io.ReadAll(r.Body)
-		require.NoError(t, err)
-		defer r.Body.Close()
-		if expectedBytes != nil {
-			require.Equal(t, expectedBytes, b)
-		}
-		r.Body = io.NopCloser(bytes.NewBuffer(b))
-		// t.Log("body", string(b))
```

Comment:
I was initially only doing the json unmarshal, but found it was useful in development to be able to log the full body in the test. But it's fine to remove this now to declutter.

Reply:
I can bring it back! just lmk

Reply:
It's all good. I can always just paste it in for dev testing later :)